### PR TITLE
fix: keybinds_hint.sh maintenance

### PR DESCRIPTION
**Pull request by kRHYME7**
# Pull Request

Adding some comprehensive help message
image

* Fix the space on the SUPER 
* For now add an option to how may lines can be rendered.

### CODE REVIEW

**Comment by dieBakterie** [comment]
I had to add the paths with an leading and ending space and $HOME to work properply.
So `kb_hint_confs=" $HOME/.config/hypr/silos.conf $HOME/.config/hypr/scratchpads.conf "` works but `kb_hint_paths="~/.config/hypr/silos.conf ~/.config/hypr/scratchpads.conf"` for it to work properly on my end. I haven't understand at first what was meant with spaces, could be only neccessary for me 'cause my fork is modified a lot.

**Comment by kRHYME7** [comment]
@dieBakterie  thanks.  got lazy to treat it as array lol. 

now you can just use the bash array conventions 

` kb_hint_conf=("file1.conf" "file2.conf")  ﯦ add a custom keybinds.conf path (add it like an array) `

File: Configs/.local/share/bin/keybinds_hint.sh [hunk 1]
```diff
@@ -352,13 +358,32 @@ if ! command -v rofi &>/dev/null; then
...
+r_width="width: ${kb_hint_width:-35em};"
+r_height="height: ${kb_hint_height:-35em};"
+r_listview="listview { lines: ${kb_hint_line:-13}; }"
+r_override="window {$r_height $r_width border: ${hypr_width}px; border-radius: ${wind_border}px;} entry {border-radius: ${elem_border}px;} element {border-radius: ${elem_border}px;} ${r_listview} "
+
+# read hypr font size
+fnt_override=$(gsettings get org.gnome.desktop.interface font-name | awk '{gsub(/'\''/,""); print $NF}')
+fnt_override="configuration {font: \"JetBrainsMono Nerd Font ${fnt_override}\";}"
+
+# read hypr theme icon
```
Comment (rubiin) [inline Configs/.local/share/bin/keybinds_hint.sh:376]:
```suggestion
# Read hypr theme icon
```

File: Configs/.local/share/bin/keybinds_hint.sh [hunk 2]
```diff
@@ -352,13 +358,32 @@ if ! command -v rofi &>/dev/null; then
...
+wind_border=$((hypr_border * 3 / 2))
+elem_border=$([ $hypr_border -eq 0 ] && echo "5" || echo $hypr_border)
+
+# TODO Dynamic scaling for text and the window >>> I do not know if rofi is capable of this
+r_width="width: ${kb_hint_width:-35em};"
+r_height="height: ${kb_hint_height:-35em};"
+r_listview="listview { lines: ${kb_hint_line:-13}; }"
+r_override="window {$r_height $r_width border: ${hypr_width}px; border-radius: ${wind_border}px;} entry {border-radius: ${elem_border}px;} element {border-radius: ${elem_border}px;} ${r_listview} "
+
+# read hypr font size
```
Comment (rubiin) [inline Configs/.local/share/bin/keybinds_hint.sh:372]:
```suggestion
# Read hypr font size
```

File: Configs/.local/share/bin/keybinds_hint.sh [hunk 3]
```diff
@@ -352,13 +358,32 @@ if ! command -v rofi &>/dev/null; then
   exit 0
 fi
 
+#? Put rofi configuration here 
+# read hypr theme border
```
Comment (rubiin) [inline Configs/.local/share/bin/keybinds_hint.sh:362]:
```suggestion
# Read hypr theme border
```

**Review by rubiin — APPROVED** [review]

**Comment by kRHYME7** [comment]
Updated the script to handle ` bindd ` explicitly added description.

**Pull request merged**